rke/k8s_version_info.go

```diff
@@ -78,6 +78,11 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRKEVersion:     "1.0.0",
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		"v1.15.12-rancher1-1": {
+			MinRancherVersion: "2.4.0",
```

Comment:
Because the [issue](https://github.com/rancher/rancher/issues/26958) this PR is for says to support `1.15.12` for `2.4.x`. If this is incorrect, I can change it. I wasn't positive what information should be driving this value.

Reply:
There is also this [issue](https://github.com/rancher/rancher/issues/27046) which says to make `1.15.12` available in Rancher `2.3.x`. I was planning on handling that in a separate PR but maybe I should be making the `MinRancherVersion=2.3.0`?

Reply:
This should be 2.3.7 so that's consistent across the branches.